modules/cert-manager-configure-cpu-memory.adoc

```diff
@@ -0,0 +1,196 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-customizing-api-fields.adoc
+
+:_content-type: PROCEDURE
+[id="cert-manager-configure-cpu-memory_{context}"]
+= Overriding CPU and memory limits for the cert-manager components
+
+After installing the {cert-manager-operator}, you can configure the CPU and memory limits from the cert-manager Operator API for the cert-manager components such as cert-manager controller, CA injector, and Webhook.
```

Comment:
I think cert-manager Operator doesnt need backticks but I see there is an attribute so does it need to be used in the instance before API? AlsoI think components and controller do need backticks, so 
```suggestion
After installing the {cert-manager-operator}, you can configure the CPU and memory limits from the {cert-manager-operator} API for the `cert-manager` components such as `cert-manager` controller, CA injector, and Webhook.
```

Reply:
IMO "cert-manager" would not be in backticks. In PascalCase typically it is --> case  dependent:"`CertManager`", case independent:"`certmanager`"- which we use in backticks.